public/js/add_tables.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+
+  var table_count = 0;
+  var table_track = {};
+
+  $('#add_table').click(function(){
+    var seats = $('#number').val();
+  });
+
+
+  /**
+  *   This function takes the provided table data and
+  *   injects it into the DOM, in the #added_tables section.
+  */
+
+  var add_table_view = function(table_data) {
```

Comment:
Identifier 'add_table_view' is not in camel case.<br>Identifier 'table_data' is not in camel case.<br>'add_table_view' is defined but never used.<br>'table_data' is defined but never used.